build.sh

```diff
@@ -70,6 +70,7 @@ fi
 
 [[ -d "${BUILD_DIR}" ]] || mkdir ${BUILD_DIR}
 
+${CONAN} remote add bincrafters https://api.bintray.com/conan/bincrafters/public-conan || true
 (cd external/qt_charts_conan && ${CONAN} create .)
```

Comment:
This IMHO not required, because it is already done in the CMakeLists.txt (above).